pkg/sitewise/datasource.go

```diff
@@ -104,32 +123,30 @@ func (ds *Datasource) invoke(ctx context.Context, _ *backend.QueryDataRequest, b
 
 func (ds *Datasource) HealthCheck(ctx context.Context, req *backend.CheckHealthRequest) error {
 
-	sw, err := ds.GetClient("") // Default region
+	sw, err := ds.getClient(ctx, "") // Default region
 	if err != nil {
 		return errors.Wrap(err, "unable to load settings")
 	}
 
 	// todo: expand health check to test permission boundaries
-	_, err = sw.ListAssetModelsWithContext(ctx, &iotsitewise.ListAssetModelsInput{MaxResults: aws.Int64(1)})
+	_, err = sw.ListAssetModels(ctx, &iotsitewise.ListAssetModelsInput{MaxResults: aws.Int32(1)})
 	return errors.Wrap(err, "unable to test ListAssetModels")
 }
 
-func (ds *Datasource) HandleInterpolatedPropertyValueQuery(ctx context.Context, req *backend.QueryDataRequest, query *models.AssetPropertyValueQuery) (data.Frames, error) {
-	sw, err := ds.GetClient(query.BaseQuery.AwsRegion)
+func (ds *Datasource) HandleInterpolatedPropertyValueQuery(ctx context.Context, _ *backend.QueryDataRequest, query *models.AssetPropertyValueQuery) (data.Frames, error) {
+	sw, err := ds.GetClient(ctx, query.BaseQuery.AwsRegion)
```

Comment:
```suggestion
	sw, err := ds.getClient(ctx, query.BaseQuery.AwsRegion)
```

I was getting a panic when running the following query without this change

<img width="721" alt="interpolated-property-values-query" src="https://github.com/user-attachments/assets/0590e2cb-69ba-496f-ac3c-0d3a536f4cb5" />